index.html

```diff
@@ -1,579 +1,1002 @@
-
-  // if you wish the publication date to be other than today, set this
-  //publishDate:  "2019-11-07",
-
-  // if there is a previously published draft, uncomment this and set its YYYY-MM-DD date
-  // and its maturity status
-  // previousPublishDate:  "1977-03-15",
-  // previousMaturity:  "WD",
-
-  // extend the bibliography entries
-  localBiblio: {
-    "LD-PROOFS": {
-      title: "Linked Data Proofs",
-      href: "https://w3c-dvcg.github.io/ld-proofs/",
-      authors: [
-        "Manu Sporny",
-        "Dave Longley"
+  <title>DID Implementation Guide v1.0</title>
+  <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
+  <!--
+      === NOTA BENE ===
+      For the three scripts below, if your spec resides on dev.w3 you can check them
+      out in the same tree and use relative links so that they'll work offline,
+     -->
+  <script class="remove" src="https://www.w3.org/Tools/respec/respec-w3c"></script>
+  <script class="remove" src="https://unpkg.com/reqlist/lib/reqlist.js"></script>
+
+  <script type="text/javascript" class="remove">
+    var respecConfig = {
+      // specification status (e.g., WD, LCWD, NOTE, etc.). If in doubt use ED.
+      specStatus: "unofficial",
+
+      // the specification's short name, as in http://www.w3.org/TR/short-name/
+      shortName: "did-imp-guide",
+      xref: ["did-core"],
+
+      // subtitle for the spec
+      subtitle:
+        "A DID developer's guide to understanding and implementing Decentralized Identifiers",
+
+      // if you wish the publication date to be other than today, set this
+      //publishDate: "2019-03-26",
+      //crEnd: "2019-04-23",
+      //implementationReportURI: "https://w3c.github.io/",
+      previousMaturity: "UNOFFICIAL",
+      previousPublishDate: "2021-01-27",
+
+      // if there is a previously published draft, uncomment this and set its YYYY-MM-DD date
+      // and its maturity status
+      // previousPublishDate:  "1977-03-15",
+      // previousMaturity:  "WD",
+
+      // extend the bibliography entries
+      localBiblio: {},
+      doJsonLd: true,
+
+      github: "https://github.com/w3c/did-imp-guide",
+      includePermalinks: false,
+
+      // if there a publicly available Editor's Draft, this is the link
+      edDraftURI: "https://w3c.github.io/did-imp-guide/",
+
+      // if this is a LCWD, uncomment and set the end of its review period
+      // lcEnd: "2009-08-05",
+
+      // editors, add as many as you like
+      // only "name" is required
+      editors: [
+        {
+          name: "Orie Steele",
+          url: "https://www.linkedin.com/in/or13b/",
+          company: "Transmute",
+          companyURL: "https://www.transmute.industries/",
+          w3cid: 109171,
+        },
+        {
+          name: "Michael Prorock",
+          url: "https://www.linkedin.com/in/mprorock/",
+          company: "mesur.io",
+          companyURL: "https://mesur.io/",
+        },
```

Comment:
There is a process requirement that editors must be members of the working group.
So we have two options: 1) Michael joins the WG, 2) Michael is listed as an author, but not as an editor